ts_app_struct/transport_scheduler/.gitignore

```diff
@@ -0,0 +1,20 @@
+# The directory Mix will write compiled artifacts to.
```

Comment:
If possible, we can have this as the default application at the top-level, with the other three being dependencies. If Elixir allows tucking in of dependencies to a sub-directory, say `deps`, then we can put `input_parser, network, api` into `deps` directory.

Done this way, the top-level looks like one single Elixir project with some dependencies.